<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1849</width>
    <height>1197</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1181</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PlatForm</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../main.png</normaloff>../../main.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="lb_ia">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(88, 160, 250, 200), stop:1 rgba(0, 0, 0, 0));
background-image: url(:/JPG/Res/IA.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(88, 160, 250, 200), stop:1 rgba(0, 0, 0, 0));
</string>
        </property>
        <property name="text">
         <string>PyQt Software Platform Project</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="main">
       <attribute name="title">
        <string>main</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,2,1" columnstretch="4,2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="1" rowspan="2">
         <widget class="QTableWidget" name="systeminfo1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableWidget::item{
vertical-align: middle;
text-align: center;
}</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QProgressBar" name="pbar1">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="lb_image">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="3">
         <widget class="QTreeWidget" name="testlist1">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 255, 255);</string>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="log">
       <attribute name="title">
        <string>log</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextEdit" name="te_log"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>bak</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="groupstate">
      <layout class="QGridLayout" name="gridLayout_10">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="lb_user_title">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>User:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_ver">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>v 1.0.0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lb_main_user">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>User</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1849</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_CSV"/>
    <addaction name="actionOpen_Result"/>
    <addaction name="actionOpen_Log"/>
    <addaction name="actionReload_Scripts"/>
    <addaction name="actionReload_CSV"/>
    <addaction name="actionClose_System"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionUser_Manage"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionZmq_Debug"/>
    <addaction name="actionTcp_Debug"/>
    <addaction name="actionSerial_Debug"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionMain_Window"/>
    <addaction name="actionEdit_Window"/>
    <addaction name="actionMotion_Window"/>
    <addaction name="actionVision_Window"/>
    <addaction name="actionSN_Window"/>
    <addaction name="actionToolBar"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuUser"/>
   <addaction name="menuTool"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: flat</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="separator"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionMainwindow"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionPause"/>
   <addaction name="actionContinue"/>
   <addaction name="separator"/>
   <addaction name="actionLoginTool"/>
   <addaction name="actionEdit"/>
   <addaction name="actionAutomation"/>
   <addaction name="separator"/>
   <addaction name="actionLog"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionReload_CSV">
   <property name="text">
    <string>Reload Sequence</string>
   </property>
   <property name="toolTip">
    <string>Reload sequence</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login System</string>
   </property>
   <property name="toolTip">
    <string>Login system</string>
   </property>
  </action>
  <action name="actionLoop_Test">
   <property name="text">
    <string>Loop Test</string>
   </property>
   <property name="toolTip">
    <string>Loop test</string>
   </property>
  </action>
  <action name="actionMain_Window">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Main Window</string>
   </property>
   <property name="toolTip">
    <string>Switch to main window</string>
   </property>
  </action>
  <action name="actionEdit_Window">
   <property name="text">
    <string>Sequence Window</string>
   </property>
   <property name="toolTip">
    <string>Switch to edit window</string>
   </property>
  </action>
  <action name="actionZmq_Debug">
   <property name="text">
    <string>Zmq Debug</string>
   </property>
   <property name="toolTip">
    <string>Zmq debug</string>
   </property>
  </action>
  <action name="actionTcp_Debug">
   <property name="text">
    <string>Tcp Debug</string>
   </property>
   <property name="toolTip">
    <string>Tcp debug</string>
   </property>
  </action>
  <action name="actionSerial_Debug">
   <property name="text">
    <string>Serial Debug</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start test</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop test</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit sequence</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionLoginTool">
   <property name="text">
    <string>Login</string>
   </property>
   <property name="toolTip">
    <string>Login system</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionAutomation">
   <property name="text">
    <string>Automation</string>
   </property>
   <property name="toolTip">
    <string>Automation debug</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionMainwindow">
   <property name="text">
    <string>Mainwindow</string>
   </property>
   <property name="toolTip">
    <string>Switch to main window</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause test</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="text">
    <string>Continue</string>
   </property>
   <property name="toolTip">
    <string>Continue test</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ToolBar</string>
   </property>
  </action>
  <action name="actionReload_Scripts">
   <property name="text">
    <string>Reload Scripts</string>
   </property>
   <property name="toolTip">
    <string>Reload scripts</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh scripts</string>
   </property>
  </action>
  <action name="actionVision_Window">
   <property name="text">
    <string>Vision Window</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
   <property name="toolTip">
    <string>log</string>
   </property>
  </action>
  <action name="actionMotion_Window">
   <property name="text">
    <string>Motion Window</string>
   </property>
  </action>
  <action name="actionOpen_CSV">
   <property name="text">
    <string>Open Sequence</string>
   </property>
  </action>
  <action name="actionOpen_Config">
   <property name="text">
    <string>Open Config</string>
   </property>
  </action>
  <action name="actionOpen_Log">
   <property name="text">
    <string>Open Log</string>
   </property>
  </action>
  <action name="actionOpen_Result">
   <property name="text">
    <string>Open Result</string>
   </property>
  </action>
  <action name="actionClose_System">
   <property name="text">
    <string>Close System</string>
   </property>
  </action>
  <action name="actionUser_Manage">
   <property name="text">
    <string>User Manage</string>
   </property>
  </action>
  <action name="actionSN_Window">
   <property name="text">
    <string>SN Window</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>start_clicked()</slot>
 </slots>
</ui>
